documentation/source/user_section/tutorials/gm-wm/gm-wm-segmentation/computing-the-wm-segmentation.rst

```diff
@@ -0,0 +1,44 @@
+Computing the white matter segmentation
+#######################################
+
+In the previous step, we used ``sct_deepseg_gm`` to segment the gray matter. However, to get the white matter, we essentially need the "inverse" volume. (In other words, the full spinal cord segmentation minus the gray matter segmentation.)
+
+Segmenting the full spinal cord
+-------------------------------
+
+In order to subtract the gray matter, we will first will need to get the  full spinal cord segmentation.
+
+.. code:: sh
+
+   sct_deepseg_sc -i t2s.nii.gz -c t2s -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : Input image.
+   - ``-c`` : Contrast of the input image.
+   - ``-qc`` : Directory for Quality Control reporting. QC reports allow us to evaluate the results slice-by-slice.
+
+:Output files/folders:
+   - ``t2s_seg.nii.gz`` : 3D binary mask of the segmented spinal cord
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/gm-wm-segmentation/io-sct_deepseg_sc.png
+   :align: center
+
+Subtracting the gray matter
+---------------------------
+
+Now that we have the spinal cord segmentation, we can subtract the gray matter and be left with the white matter.
+
+.. code:: sh
+
+   sct_maths -i t2s_seg.nii.gz -sub t2s_gmseg.nii.gz -o t2s_wmseg.nii.gz
+
+:Input arguments:
+   - ``-i`` : Input image. (The full segmentation of the spinal cord.)
+   - ``-sub`` : Flag to invoke the "subtract" functionality of ``sct_maths``, subtracting ``t2s_gmseg.nii.gz`` from the input image.
+   - ``-o`` : The filename of the output image.
+
+:Output files/folders:
+   - ``t2s_wmseg.nii.gz`` : An image file containing the segmentation for the white matter.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/gm-wm-segmentation/io-sct_maths_gm_wm.png
```

Comment:
Unlike gray matter and full spinal cord segmentation, I am unable to get QC report for white matter on broswer.

Terminal output :

```(venv_sct) ➜  t2s git:(sy/test-sct-tuts) ✗ sct_maths -i t2s_seg.nii.gz -sub t2s_gmseg.nii.gz -o t2s_wmseg.nii.gz

Spinal Cord Toolbox (git-sy/test-sct-tuts-30e649037ab929937944d0df064b7a5cee32fb04)

sct_maths -i t2s_seg.nii.gz -sub t2s_gmseg.nii.gz -o t2s_wmseg.nii.gz

File t2s_wmseg.nii.gz already exists. Will overwrite it.
```
Browser (`file:///home/srishtiy/qc_singleSubj/index.html`): 

![image](https://user-images.githubusercontent.com/23316280/133835581-b1785b9a-c1a9-4471-a21d-d52059850787.png)

Reply:
Good question! The QC report will only be generated for commands that use the `-qc` argument. And in this case, `sct_maths` doesn't have a `-qc` option.

That could definitely be made more clear in the "Before Starting" section. What do you think of this wording update?

* Before: "After some steps in this tutorial, instructions are provided to open the output images using Quality Control (QC) reports and FSLeyes.
* After: "For commands that support the `-qc` option, instructions will be provided to inspect the output images using the Quality Control (QC) report interface. If you have FSLeyes installed, some commands will also print instructions to open the output images using FSLeyes."